### PR TITLE
Add 'minimum width' property

### PR DESCRIPTION
**Pull request by vvit**
Hi there, great repo. It seems there are couple of requests for this feature: #32 #33
The user can set a minimal width so the shape will always be circle, not oval/ellipse.
What do you think?
Thanks.

### CODE REVIEW

File: JSBadgeView/JSBadgeView.h [hunk 1]
```diff
@@ -88,6 +88,11 @@ typedef NS_ENUM(NSUInteger, JSBadgeViewAlignment)
 @property (nonatomic, assign) CGRect frameToPositionInRelationWith UI_APPEARANCE_SELECTOR;
 
 /**
+ * The minimal width of a badge circle. We need this to avoid elipse shapes when using small fonts.
```
Comment (JaviSoto) [inline JSBadgeView/JSBadgeView.h]: s/minimal/minimum?

**Comment by JaviSoto** [comment]
Thanks for the PR!

I'm wondering: is this change necessary? What if you apply this logic to the layout of the parent view, by making sure you size the `JSBadgeView` by applying a minimum width?

**Comment by vvit** [comment]
@JaviSoto thanks. The problem is whenever I call something like this:
```
if (_badgeView.badgeText.length == 1)
{
    CGRect rc = _badgeView.frame;
    rc.size = CGSizeMake(30.f, _badgeView.frame.size.height);
    _badgeView.frame = rc;
}
```
JSBadgeView::layoutSubviews recalculates the frame so it's set to previous state which is ellipse.

File: JSBadgeView/JSBadgeView.h (hunk 1 above)
Comment (JaviSoto) [inline JSBadgeView/JSBadgeView.h]: Do you mind fixing this comment? :)

**Comment by JaviSoto** [comment]
Yeah, I think this is good :) Sorry for taking so long!
Do you mind fixing that minor comment and I'll merge this? :)

**Comment by vvit** [comment]
@JaviSoto pls check the latest commit. Thx a lot!

**Comment by JaviSoto** [comment]
:+1: Thank you!!

**Pull request merged**